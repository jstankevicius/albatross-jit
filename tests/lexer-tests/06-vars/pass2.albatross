int name
char of
string many
int[] variables
