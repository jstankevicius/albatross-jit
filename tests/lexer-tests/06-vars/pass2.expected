1 1 TYPE int
5 1 NAME name TYPE int
1 2 TYPE char
6 2 NAME of TYPE char
1 3 TYPE string
8 3 NAME many TYPE string
1 4 TYPE int
4 4 [
5 4 ]
7 4 NAME variables TYPE int
